plugin_manager.py

```diff
@@ -808,7 +810,7 @@ def latest_compatible_version(self):
                     break
         if self._latest_compatible_version is None:
             raise NoCompatibleVersion(
-                f"{self.name} has no version compatible with API {babase.app.api_version}."
+                f"{self.name} has no version compatible with API {babase.app.api_version if build_number < 21282 else babase.app.env.api_version}."
```

Comment:
this'll introduce lots of code changes that'll become legacy code in a few months (adding on to maintainence burden), let's add a compatibility layer instead, something like this:

```python
import babase
from baenv import TARGET_BALLISTICA_BUILD

if TARGET_BALLISTICA_BUILD < 21282:
    babase.app.env.api_version = babase.app.api_version
    babase.app.env.debug_build = babase.app.debug_build
    

# Now let's use `babase.app.env.*` everywhere else
# in code irrespective of whatever build version user is on.
```
then once a couple months have passed and most ppl are on newer versions, we can then simply nuke this compatibility layer code all at once easily.

Reply:
You can do those changes my laptop is dead

Reply:
ow, don't worry i'll take this battle up, get well soon lappy!